src/parser/content-evaluator-module.ts

```diff
@@ -15,6 +15,9 @@ import {
 import { BaseModule } from "../types/module";
 import { ContextPlugin } from "../types/plugin-input";
 import { GithubCommentScore, Result } from "../types/results";
+import { TfIdf } from "../helpers/tf-idf";
+
+const TOKEN_MODEL_LIMIT = 124000;
```

Comment:
This depends on the model and possibly should be an environment variable because we might change models.

Reply:
https://github.com/ubiquity-os-marketplace/text-conversation-rewards/pull/225/commits/a0695e3d6721576b97f03b33a2cc7314f3da3d95

Reply:
Hard coding the 12400 doesn't seem like a solution there either

Reply:
@0x4007 It is not hard coded but configurable within the config file?
There is no API to retrieve a model max token value as far as I know.

Reply:
Line 179 is hard coded

Reply:
What should I do if the configuration is undefined, should I throw an error and stop the run then?

Reply:
Yes if we don't have it saved in our library or collection of known amounts then it should throw